lib/contentProfileListener.js

```diff
@@ -79,6 +79,25 @@ class ContentProfileImporter extends SHRContentProfileParserListener {
     this._currentNs = ctx.namespace().getText();
   }
 
+  enterContentDef(ctx) {
+    // Check to see if Entry/Group has flags. Currently only NP
+    if (ctx.headerFlags()) {
+      const pathStr = ctx.contentHeader().simpleName().getText();
+      const identifier = new Identifier(this._currentNs, pathStr);
+      this._currentDef = new ContentProfile(identifier);
+      this._currentDef.grammarVersion = this._currentGrammarVersion;
+      // NP should be used at top level, so no need to find path
+      this._currentRule = new ContentProfileRule([this._currentDef.identifier]);
```

Comment:
I don't think we need to pass in a path here, and we actually probably shouldn't. This rule is currently attached to the content profile, which has the currently entry's identifier associated with it. Since we only care about the entry's behavior itself, and not any path associated with it, we should be able to remove the path argument entirely.